apps/examples/wc-angular12/package.json

```diff
@@ -27,13 +27,6 @@
         "@angular-devkit/build-angular": "15.2.6",
         "@angular/cli": "15.2.6",
         "@angular/compiler-cli": "15.2.0",
-        "@types/jasmine": "4.3.0",
```

Comment:
This change was done to remove unneeded dependencies, given we currently don't unit test Angular.

The main reason for this, was due to the fact I want to make the Vitest types available globally, and given most test libraries follow the `describe` / `it` / `expect` convention, the addition of `@types/jasmine` was causing type conflicts.